CHANGELOG.md

```diff
@@ -135,7 +135,6 @@ Here are the detailed release notes for this release (**note that ❗ denotes a
     - Remove `@oclif/command`
     - Remove `@oclif/config`
     - Update `archiver` 5.3.1 to 7.0.1
-    - Update `gulp-prettier` from 4.0.0 to 5.0.0
```

Comment:
This didn't actually make it into [the v16 PR](https://github.com/zapier/zapier-platform/pull/926), removing it to prevent future confusion